server/controllers/routes/admin/cohort/editCohort.js

```diff
@@ -0,0 +1,20 @@
+const { putSpecificCohort } = require('../../../../database/queries');
+const { editCohortSchema } = require('../../../../validation/index');
+
+exports.editCohort = async (req, res) => {
+  try {
+    await editCohortSchema.validate(req.body);
```

Comment:
If you test this ```editCohortSchema.validate(req.body)``` by default it's aborts all the validations steps once it hit the first validation error , to get all error you would just add the optional second parameter``` {abortEarly: false }``` to the call  to ```editCohortSchema.validate(req.body)``` which look like as I suggested :

```suggestion
    await editCohortSchema.validate(req.body, {abortEarly: false });

```